info.beyondallreason.bar.appdata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>info.beyondallreason.bar</id>
+    <name>Beyond All Reason</name>
+    <summary>The Total Annihilation Inspired RTS you've been waiting for</summary>
+    <metadata_license>CC-BY-SA 4.0, CC-BY-NC-ND 4.0, and others</metadata_license>
```

Comment:
this is probably not what you think. This is the license of THIS file. Prefered license for it is `CC0-1.0`

Reply:
I did mix that up with the game assets. Thanks for noticing